specification/iotoperations/IoTOperations.Management/models/instance.tsp

```diff
@@ -52,33 +52,14 @@ model InstanceProperties {
   version?: string;
 
   @doc("The reference to the Schema Registry for this AIO Instance.")
-  @pattern("^[a-z0-9][a-z0-9-]*[a-z0-9]$")
-  schemaRegistryNamespace: string;
-
-  @doc("The configuration for components of the AIO Instance.")
-  components?: Components = #{};
+  schemaRegistryRef: SchemaRegistryRef;
 }
 
-@doc("The configuration for components of the AIO Instance.")
-model Components {
-  @doc("The properties of the ADR instance.")
-  adr?: AdrProperties = #{ state: OperationalMode.Enabled };
-
-  @doc("The properties of the AKRI instance.")
-  akri?: AkriInstanceProperties = #{ state: OperationalMode.Enabled };
-
-  @doc("The properties of the connectors instance.")
-  connectors?: ConnectorInstanceProperties = #{
-    state: OperationalMode.Enabled,
-  };
-
-  @doc("The properties of the dataflows instance.")
-  dataflows?: DataflowInstanceProperties = #{ state: OperationalMode.Enabled };
-
-  @doc("The properties of the device provisioning instance.")
-  schemaRegistry?: SchemaRegistryInstanceProperties = #{
-    state: OperationalMode.Enabled,
-  };
+@doc("The reference to the Schema Registry for this AIO Instance.")
+model SchemaRegistryRef {
+  @doc("The resource ID of the Schema Registry.")
+  @visibility("read")
```

Comment:
I don't think visibility should be set to read here. We want this property to be writable, updatable and readable